client/src/main.rs

```diff
@@ -137,7 +137,7 @@ fn repl(rt: Runtime, mut fsc: FlightServiceClient<Channel>) -> Result<()> {
     let mut editor = Editor::<ClientHelper>::new()?;
     let table_names = retrieve_table_names(&rt, &mut fsc)?;
     editor.set_helper(Some(ClientHelper::new(table_names)));
-    let history_file_name = ".mmdbc_history";
+    let history_file_name = ".mdb_history";
```

Comment:
Should it be `mdbc_history`?

Reply:
The client `mmdbc` was previously renamed to `mmdb` to follow the convention on Unix-like operating systems to use `program-name` for clients and `program-name-d` for [Daemons](https://en.wikipedia.org/wiki/Daemon_(computing)). For example `sshd` / `ssh`, `telnetd` / `telnet`, and `influxd` / `influx`. However, for some reason ".mmdbc_history" was missed when `mmdbc` was renamed to `mmdb`, so it was purposely renamed to ".mdb_history" to be consistent with the name of the binary.